portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase

```diff
@@ -448,6 +448,134 @@ definition {
 			fieldValue = "Text ES");
 	}
 
+	@description = "Verify that the Date field is displayed in the same language selected by the user on the Entries page."
+	@priority = 4
+	test CheckDateFieldResponsesSameLanguageOnTheEntriesPage {
+		property portal.upstream = "true";
```

Comment:
Removed property portal.acceptance = "true";

Reply:
@kristinonias I don't think we need this since it is already defined at the top